### PR TITLE
chore: fix all typos on the repository

### PR DESCRIPTION
**Pull request by hoangdv2429**
This PR will fix typos that exist on current repository 
(variable name will stay untouch)

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords
- [x] I reviewed file changes myself.

### CODE REVIEW

File: .gitignore
Comment (hoangdv2429) [inline .gitignore:1]: this one is left over from my previous PR.

**Comment by hoangdv2429** [comment]
@rootulp Hi sir, can you review this one ?

File: test/testground/manifest.toml [hunk 1]
```diff
@@ -38,7 +38,7 @@ blobs_per_sequence = { type = "int", default = 1 }
 inbound_peer_count = { type = "int", default = 40 }
 outbound_peer_count = { type = "int", default = 10 }
 gov_max_square_size = { type = "int", default = 256 }
-max_block_bytes = { type = "int", deafult = 100000000 }
+max_block_bytes = { type = "int", default = 100000000 }
```
Comment (rootulp) [inline test/testground/manifest.toml:41]: wow I'm surprised the previous incorrect spelling didn't result in broken behavior

**Review by rootulp — APPROVED** [review]
Thanks for the contribution @hoangdv2429 !

File: test/testground/manifest.toml (hunk 1 above)
Comment (hoangdv2429) [reply test/testground/manifest.toml:41]: I was surprised too, maybe max_block_bytes is not being test.. ?

**Pull request merged**